### PR TITLE
dev: add dm 1.0-ga benchmark

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? 

Add dm 1.0-GA benchmark

### What is the related PR or file link(s)? 
N/A

### Which version does your change affect? 

dev

### CODE REVIEW

File: dev/benchmark/dm-v1.0-ga.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+| 172.16.4.42 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+| 172.16.4.43 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+| 172.16.4.44 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+
+Hardware information:
+
+| Type         |                                                    |
+| ------------ | -------------------------------------------------- |
+| CPU          | 40 CPUs, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz |
+| Memory       | 188G                                               |
```
Comment (WangXiangUSTC) [inline dev/benchmark/dm-v1.0-ga.md]: every machine has 188G memory?

File: dev/benchmark/dm-v1.0-ga.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+Others:
+
+* network rtt between servers: rtt min/avg/max/mdev = 0.074/0.088/0.121/0.019 ms
+
+### Cluster topology
+
+| Machine IP  | Deployment instance                |
+| ----------- | ---------------------------------- |
+| 172.16.4.39 | PD1, MySQL1, DM-worker1, DM-master |
+| 172.16.4.40 | PD2, MySQL2, DM-worker2            |
```
Comment (WangXiangUSTC) [inline dev/benchmark/dm-v1.0-ga.md]: seems the MySQL2 is not used

File: dev/benchmark/dm-v1.0-ga.md (hunk 1 above)
Comment (amyangfei) [reply dev/benchmark/dm-v1.0-ga.md]: yep

File: dev/benchmark/dm-v1.0-ga.md (hunk 2 above)
Comment (amyangfei) [reply dev/benchmark/dm-v1.0-ga.md]: Unused service removed

**Comment by amyangfei** [comment]
PTAL @GregoryIan @csuzhangxc @WangXiangUSTC

File: dev/benchmark/dm-v1.0-ga.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+| 172.16.4.39 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+| 172.16.4.40 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+| 172.16.4.41 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+| 172.16.4.42 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+| 172.16.4.43 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+| 172.16.4.44 | CentOS Linux release 7.6.1810 | 3.10.0-957.1.3.el7.x86_64 | ext4             |
+
+Hardware information:
+
+| Type         |                                                    |
```
Comment (csuzhangxc) [inline dev/benchmark/dm-v1.0-ga.md]: miss the header for the second column

File: dev/benchmark/dm-v1.0-ga.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+| load pool size | latency of execution txn (s) | time (s) | speed (MB/s) | TiDB 99 duration (s) |
+| :------------: | :--------------------------: | :------: | :----------: | :------------------: |
+| 2              | 0.250                        | 425.9    | 9.1          | 0.23                 |
+| 4              | 0.523                        | 360.1    | 10.7         | 0.41                 |
+| 8              | 0.986                        | 267.0    | 14.5         | 0.93                 |
+| 16             | 2.022                        | 265.9    | 14.5         | 2.68                 |
+| 32             | 3.778                        | 262.3    | 14.7         | 6.39                 |
+| 64             | 7.452                        | 281.9    | 13.7         | 8.00                 |
+
+#### Benchmark result with different sql in per statement in dump file
```
Comment (csuzhangxc) [inline dev/benchmark/dm-v1.0-ga.md]: how about `with different row count in per statement`?

File: dev/benchmark/dm-v1.0-ga.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+|            310             |  -s 62500 -r 320000  |            0.413             |  322.6   |     12.0     |         0.49         |
+
+### Increase replication benchmark case
+
+#### Test procedure
+
+- Set up environment
+- Use sysbench to create the table and generate the initial data in upstream MySQL
+- Start DM-task in the `all` mode, and wait task enters `sync` unit
+- Use sysbench to generate incremental data in upstream MySQL and observe grafana metrics and DM `query-stats`
```
Comment (csuzhangxc) [inline dev/benchmark/dm-v1.0-ga.md]:
```suggestion
- Use sysbench to generate incremental data in upstream MySQL and observe grafana metrics and DM `query-status`
```

File: dev/benchmark/dm-v1.0-ga.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+
+## Recommended parameters
+
+### dump unit
+
+we recommend the statement size between 5KB and 1MB, and row count in one statement is approximately 1000-5000, which is based on the row size in your scenario.
+
+### load unit
+
+we recommend to set pool-size to 16
```
Comment (csuzhangxc) [inline dev/benchmark/dm-v1.0-ga.md]:
```suggestion
we recommend to set `pool-size` to 16
```
ditto for L197

File: dev/benchmark/dm-v1.0-ga.md [hunk 7]
```diff
@@ -29,7 +29,7 @@ System information:
 
 Hardware information:
 
-| Type         |                                                    |
+| Type         | Parameter                                          |
```
Comment (csuzhangxc) [inline dev/benchmark/dm-v1.0-ga.md]: "Specification"?

**Pull request merged**